core/src/test/resources/configs/unified/8.3.xml

```diff
@@ -0,0 +1,262 @@
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:8.3 http://www.infinispan.org/schemas/infinispan-config-8.3.xsd"
+      xmlns="urn:infinispan:config:8.3">
```

Comment:
Did you intend to add these 8.x.xml files, or did they slip in when testing the script?

Reply:
It is intentional